src/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -7,6 +7,33 @@ endif()
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 include (GenerateExportHeader)
 
+if($ENV{__BuildArch} STREQUAL x86 OR $ENV{__BuildArch} STREQUAL x64)
+set(NATIVECOMPRESSION_SOURCES
+    clrcompression.def
+    zlib-intel/adler32.c
```

Comment:
Also, are all of these files used? I know for zlib-adler there were a bunch of files that weren't getting used in the build so we commented out their inclusion. Can we do that here?